Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+ESDC prefers **open source software** and products that adopt **open standards** over proprietary ones.
+The adoption of Open source software is the main vendor locking mitigation strategy.
+The adoption of open standards is the main interoperability enabler strategy.
+
+Legacy technologies like mainframe COBOL have been replaced by using a strangled approach as opposed to a big bang one.
+The transition of legacy systems into modern ones required large investments on the business and data side, as opposed to IT, in order to accurately depict business rules and processes agnostically of IT systems.
+
+ESDC's legacy core systems are modernized into a set of micro services as opposed to monolithic systems, each working together as an aggregate to bring Pension, Employment Insurance, and Old Age Security delivery to citizens.
+The number of services (or number of systems) is not a concern, the concern is rather the ability to effectively manage technical debt and keep up with demands by making timely, small iterative changes.
+
+ESDC leverages different networks based on its need.
+The Internet for its most common day to day use, reducing bottleneck to its corporate and production network.
+The corporate network for accessing corporate systems (e.g. HR, Finance, and emails), and production network for accessing the systems hosting Canadians' data and proving services to Canadians.
+
+The CIO is seen as a critical partner in service design and delivery, having a seat at the table as much as the Chief Data Office (CDO) and the Service's strategic direction.
+IT's credibility to deliver has been achieved by automating much of its internal development processes when it adopted DevOps.
+
+## 4. Modern tech, technology practices and information stewardship
+
+IT's innovation is focused on solving ESDC's business problems.
+Technologies are experimented with to solve business problems, improving the relationships between IT and Service personnel.
+All of ESDC's IT innovation is released as open source on the Internet to provide its benefits to other federal departments, jurisdictions, and the public.
+
+ESDC enables and promotes innovation in order to keep up with the fast pace of technology changes.
+This is achieved by:
+
+- managing risks in a bi-modal way, allowing space for mode 2 (experiments and innovation) to operate, where a mode 1 (rigorous and based on predictability) is used for production and mission critical systems only
+- providing stable funding to manage support and infrastructure for innovators
+- having well defined processes to provide support in scaling successful experiments into production
+- prioritizing innovation in the organization, using metrics focusing on business objectives, improved pace, and success rates
```

Comment:
prioritizing innovation in the organization ==> supporting innovation in the organization 

We need to create a clear process by which innovative grass roots projects can end up in production. Many innovation maturity models speak to innovation products being able to deliver value to the business as a sign of innovative maturity.

Reply:
Add another bullet like "inspiring a culture of innovation to its employees" ?